css/src/tokens/palette.scss

```diff
@@ -348,4 +348,9 @@ $palette-yellow-high-contrast: #ff0 !default;
 $palette-yellow-high-contrast-hover: #ff3 !default;
 $palette-visited-high-contrast: #3cff00 !default;
 
+// Ask Learn
+$palette-ask-learn-yellow: #ffb152 !default;
```

Comment:
Another "fun" thing about gradients - have these colors being tested for a11y color contrast requirements? Down the road, it can be used on text, and it should not fail the contras ratio.

Reply:
we are good on color contrast now since we're just using a red and a purple, both of which are higher than 3:1 contrast ratio on light and dark backgrounds. I removed the yellow